src/encoded/static/components/item-pages/QualityMetricView.js

```diff
@@ -109,68 +96,284 @@ class QualityMetricViewOverview extends React.PureComponent {
+                            {qcMetricsSummary}
+                        </div>
+                    </WrapInColumn>) : null}
                 <WrapInColumn wrap="col-12 col-md-9" defaultWrapClassName="col-sm-12">
                     <div className="inner">
                         <object.TooltipInfoIconContainerAuto result={null} property={null} tips={tips}
-                            elementType="h5" fallbackTitle="Summary" />
-                        {metricsSummary}
+                            elementType="h4" fallbackTitle="All Metrics" className="qc-section-title" />
+                        {qcMetrics}
                     </div>
                 </WrapInColumn>
-                <WrapInColumn wrap="col-12 col-md-9" defaultWrapClassName="col-sm-12">
-                    <div className="inner">
-                        <object.TooltipInfoIconContainerAuto result={context} property={null} tips={tips}
-                            elementType="h5" fallbackTitle="All Metrics" />
-                        {metrics}
+            </React.Fragment>
+        );
+    }
+}
+
+class QCMetricFromEmbed extends React.PureComponent {
+    
+    static percentOfTotalReads(quality_metric, field){
+        var numVal = object.getNestedProperty(quality_metric, field);
+        if (numVal && typeof numVal === 'number' && quality_metric && quality_metric['Total reads']){
+            var percentVal = Math.round((numVal / quality_metric['Total reads']) * 100 * 1000) / 1000;
+            var numValRounded = valueTransforms.roundLargeNumber(numVal);
+            return (
+                <span className="inline-block" data-tip={"Percent of total reads (= " + numValRounded + ")."}>{ percentVal + '%' }</span>
+            );
+        }
+        return '-';
+    };
+    
+    constructor(props) {
+        super(props);
+
+        const { defaultOpen } = props;
+        this.toggleOpen = _.throttle(this.toggleOpen.bind(this), 1000);
+        this.state = {
+            'open': defaultOpen || false,
+            'closing': false
+        };
+    }
+
+    toggleOpen(open, e){
+        this.setState(function(currState){
+            if (typeof open !== 'boolean'){
+                open = !currState.open;
+            }
+            var closing = !open && currState.open;
+            return { open, closing };
+        }, ()=>{
+            setTimeout(()=>{
+                this.setState(function(currState){
+                    if (!currState.open && currState.closing){
+                        return { 'closing' : false };
+                    }
+                    return null;
+                });
+            }, 500);
+        });
+    }
+
+    render() {
+        const { metric, qcProperty, schemaItem, schemas, fallbackTitle, tips, percent } = this.props;
+        const { open, closing } = this.state;
+
+        if (schemaItem && typeof schemaItem.qc_order !== 'number') return null;
```

Comment:
Also probably good place for JS destructuring (subjective), e.g. could have:
```javascript
const { qc_order = null, title = null, description: tip = null } = schemaItem;
if (typeof qc_order !==' number') return null;     // Less text!
let value = metric[qcProperty];
if (typeof value === "undefined") return null;

let subQCRows = null;
if (qcProperty !== 'qc_list' && schemaItem && typeof schemaItem === 'object') {
...
```

Reply:
thanks @alexkb0009. refactored the code.